pdl/protocol.pdl

```diff
@@ -1662,4 +1662,7 @@ domain Internal
       string lastScreenData 
       string lastScreenMimeType  
 
-
+  # Set the user's current access token
+  command setAccessToken
```

Comment:
How would you feel about making a new `Authentication` domain for this? Currently we only send `Internal.*` commands from the driver itself, or other internal processes. Given that, I don't think we'd want the DevTools to send any `Internal.*` message.

Reply:
Done.